packages/cli/src/modules/insights/__tests__/insights.service.test.ts

```diff
@@ -779,3 +786,72 @@ describe('timers', () => {
 		expect(mockPruningService.stopPruningTimer).toHaveBeenCalled();
 	});
 });
+
+describe('TransactionNotStarted error on sqlite without pooling', () => {
```

Comment:
I'm not sure I understand what this test is for. The description says something should throw. The assertions just check that all promises resolve.